tests/storages_tests/test_with_server.py

```diff
@@ -122,10 +119,10 @@ def test_store_infinite_values(input_value: float, expected: float, storage_url:
         storage = optuna.storages.RDBStorage(url=storage_url)
     study_id = storage.create_new_study()
     trial_id = storage.create_new_trial(study_id)
-    storage.set_trial_intermediate_value(trial_id, 1, input_value)
-    storage.set_trial_state_values(trial_id, state=TrialState.COMPLETE, values=(input_value,))
-    assert storage.get_trial(trial_id).value == expected
```

Comment:
I added tests for it also.

Reply:
Hmm, it seems to be a kind of anti-pattern called [conditional test logic](http://xunitpatterns.com/Conditional%20Test%20Logic.html).
In addition, refactoring and code fix should be done in separately because we can easily confirm the code fix does not break the current behavior.  Could you keep the current test function?

Reply:
Done.

Reply:
Thanks!